posydon/visualization/plot_pop.py

```diff
@@ -251,4 +258,137 @@ def plot_hist_properties(x, df_intrinsic=None, df_observable=None,
     if path:
         plt.savefig(path)
     if show:
-        plt.show()
\ No newline at end of file
+        plt.show()
+        
+        
+def plot_popsyn_over_grid_slice(pop, grid_type, met_Zsun, slices=None, channel=None, 
+                                plot_dir='./', prop=None, prop_range=None, 
+                                log_prop=False, alpha=0.3, s=5., 
+                                show_fig=True, save_fig=True, close_fig=True,
+                                verbose=False):
+
+    # load grid
+    met = convert_metallicity_to_string(met_Zsun)
+    grid_path = os.path.join(PATH_TO_POSYDON_DATA, f'POSYDON_data/{grid_type}/{met}_Zsun.h5')
```

Comment:
Shouldn't `PATH_TO_POSYDON_DATA` already include the directory `POSYDON_data`?

Reply:
no, it does not. We made this choice in v1 to be the working directory to where the `POSYDON_data` tar file will be downloaded from Zenodo and the uncompressed to the `POSYDON_data` directory. Though I agree this is confusing, once we revisit how to store and import paths from a `config.py` file reading `.env` we can edit this.